heartbeat/_meta/fields.common.yml

```diff
@@ -58,6 +58,28 @@
           description: >
             Indicator if monitor could validate the service to be available.
 
+        - name: check_group
+          type: keyword
+          description: >
+            A token unique to a simultaneously invoked group of checks as in the case where multiple IPs are checked for a single DNS entry.
+
+- key: summary
+  title: "Monitor summary"
+  description: "Present in the last event emitted during a check. If a monitor checks multiple endpoints, as is the case with `mode: all`."
+  fields:
+    - name: status
```

Comment:
I'm wondering if we really need to ship this field or leave it to the UI to calculate and make a call. It could be that someone decides if 4/5 are up, status should be green.

Reply:
It's a good question. For a minute I too thought we shouldn't have it. However, I realized it'd be useful for people building dashboards in kibana with terms aggregations.

One thought, we could add a third `degraded` state. WDYT?

Reply:
I think it's ok if that is something that only the UI can have and cannot be done in dashboards.